python/HeaderService/hslib_salobj.py

```diff
@@ -427,7 +427,23 @@ def collect_from_HeaderService(self):
             self.metadata['RAEND'] = ra
             self.metadata['DECEND'] = dec
         else:
-            self.log.info("No 'ELEND' and 'AZEND'")
+            self.log.info("No 'ELEND' and 'AZEND' to compute RA/DEC END")
+
+        if 'RA' and 'DEC' and 'ROTPA' in self.metadata:
+            # --- TODO ----
+            # In the near future, once radesys comes from telemetry,
+            # we should use self.metadata['RADESYS'] instead of the
+            # hardcorded FK5 value we pluged here.
+            # --------------
```

Comment:
What's the ticket number for adding `RADECSYS`?

Reply:
Couldn't find it with my jira knowledge. Will keep trying.

Reply:
https://github.com/lsst-dm/HeaderService/pull/34/files

Reply:
That's not a ticket number.  Is it DM-6576?

Reply:
Since the last review, `RADECSYS` has been included `ATMCS_logevent_target.radecsys` since ts_xml 4.4.1. So, I will use `self.metadata['RADESYS']` that it is capturing `ATMCS_logevent_target.radecsys`. However, like a few of the keywords expected by TSS it might not be populated yet. We will need to continue to audit the header for completeness.